File: 프로그래머스/0/181851. 전국 대회 선발 고사/전국 대회 선발 고사.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

int solution(vector<int> rank, vector<bool> attendance) {
    int answer = 0;
    
    vector<pair<int,int>> count;
    for(int i=0;i<rank.size();i++)
    {
        if(attendance[i]==true)
            count.push_back({rank[i],i});
    }
    sort(count.begin(),count.end());
    answer=10000 * count[0].second + 100 * count[1].second + count[2].second;
    
    
    return answer;
}

File: 프로그래머스/0/181851. 전국 대회 선발 고사/전국 대회 선발 고사.py
def solution(rank, attendance):
    answer = 0
    
    student=[[rank[i],i] for i in range(len(rank)) if attendance[i]==True]
    
    student.sort()
    a=student[0][1]
    b=student[1][1]
    c=student[2][1]
    
    answer=10000*a+100*b+c
    
    return answer